### DISCUSSION TITLE
rasterizing : line thickness

### DISCUSSION THREAD

**Discussion by TekayaNidham** [mozman/ezdxf#638] [Q&A]
Is it possible to reduce lines thickness 
### png vs dxf : 
![2022-02-24_15-02](https://user-images.githubusercontent.com/62834628/155538632-675b0d07-9bcf-41df-a1d3-791b24f1d4b0.png)

code i used : 
```
if not auditor.has_errors:
    fig = plt.figure()
    ax = fig.add_axes([0, 0, 1, 1])
    ctx = RenderContext(doc)
    ezdxf.addons.drawing.properties.MODEL_SPACE_BG_COLOR = "#FFFFFF"
    out = MatplotlibBackend(ax)
    Frontend(ctx, out).draw_layout(doc.modelspace(), finalize=True)
    fig.savefig('your.png', dpi=300)
```

**Comment by mbway** [comment]
you could try increasing the dpi?

**Reply by TekayaNidham** [reply]
tried it even with 1200, still the same, better resolution thick line 😄

**Reply by TekayaNidham** [reply]
update : i deleted the frame element and cropped the floor plan and it somehow solved it
I don't know how CAD files or dxf-ezdxf workflow works but if someone sees this discussion maybe explain it ?

**Reply by TekayaNidham** [reply]
### Before vs after
![image](https://user-images.githubusercontent.com/62834628/155693890-37401b49-e587-41a8-b8bd-d3dcaf0433f8.png)
